### ISSUE TITLE
Add option to gFind, gAperture, and gMap that will allow user to specify a server to query.

### ISSUE THREAD

**Issue by scfleming** [cmillion/gPhoton#187]
For testing purposes the team at MAST need the option to easily change the server being used to query from the client itself.  The easiest way to do this is to introduce an optional parameter to the Big Three(™) that all public users will never provide, but the testers at MAST will be able to use, e.g.:

x = gFind(skypos=[12.34, 56.78], server="some_server_url_the_testers_know_about")

If possible can we do this as a hotfix to the master branch before starting 1.27.0?

**Comment by cmillion** [comment]
Ok. Let me look into it. It might be more complicated than a hotfix because it drives right to the heart of gQuery. But I have at least one idea for how to simplify it.

**Comment by scfleming** [comment]
Yeah my initial thought was to pass the parameter from the Big Three down to gQuery, but that might involve a LOT of passing down the chain now that I think about it...
